docs-js/langchain/openai.mdx

```diff
@@ -186,6 +186,33 @@ Use the `bindTools()` method to define the set of tools a model can access.
 For more details, see the [official LangChain documentation on tool binding](https://js.langchain.com/docs/concepts/tool_calling/#tool-binding).
 For a usage example, refer to the [getting started with agents tutorial](../tutorials/getting-started-with-agents/langchain#define-mock-tools).
 
+### Structured Output
+
+It is often useful to have a model return output that matches a specific schema.
+This schema can be defined using either a [Zod](https://zod.dev/) schema or an OpenAI-style JSON schema.
+For more details on structured output, refer to the [official LangChain documentation on structured output](https://js.langchain.com/docs/modules/model_io/output_parsers/structured_output).
+Below is an example using `json_schema` response type and passing in a Zod schema.
+
+```ts
+import { z } from 'zod/v4';
```

Comment:
```suggestion
import * as z from 'zod';
```
[pp] Using the format from zod official website to keep the usage consistent as other occurrences in our sample-code and docs.